src/cli/cli.ts

```diff
@@ -400,3 +417,9 @@ function validateOptions(options: Options) {
     process.exit(0);
   }
 }
+
+function resolveHome(filepath: string | undefined) {
+  if (filepath && filepath[0] === '~')
```

Comment:
I'm very confused by this. Shouldn't bash do this for us?

Reply:
Don't know, I just copied it. Let me check.

Reply:
Removed.